sgrep_lint/sgrep.py

```diff
@@ -62,11 +62,24 @@
         help=f"only invoke sgrep if config(s) are valid",
         action="store_true",
     )
+
+    parser.add_argument(
+        "--include",
+        action="append",
+        default=[],
+        help="Scan only files with this filename, such as --include='*.js' Add multiple times to include multiple patterns.",
```

Comment:
@ievans can you check these examples? ☺️

Reply:
It's an improvement! but could we a bit more explicit about what exactly is matched? Eg compare with man grep:
```
     --exclude
             If specified, it excludes files matching the given filename pattern from the search.  Note that --exclude patterns take priority over --include patterns, and if no --include pattern is specified, all files are searched that are not excluded.  Patterns are matched to the full path specified, not only to the filename component.
```
Is that last bit true here or no?